proposals/reward-proposal-v2-reward-calculation.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Epoch Reward Calculation
```

Comment:
```suggestion
title: Epoch Reward Calculation V2
```

Reply:
I also feel that the title and filename (in kebab case?) should match. Same for the the other file in this PR